ods_ci/tests/Tests/1100__data_science_pipelines/1110__test-run-data-science-pipelines-operator-e2e-tests.robot

```diff
@@ -24,29 +23,24 @@ Run Data Science Pipelines Operator Integration Tests
     [Tags]
     ...     DataSciencePipelines-Backend
     ...     Tier1
-    ...     ODS-2632    AutomationBug
+    ...     ODS-2632
     ${openshift_api}    Get Openshift Server
     Log    ${openshift_api}
-    ${return_code}    ${output}    Run And Return Rc And Output    cd ${DATA-SCIENCE-PIPELINES-OPERATOR-SDK_DIR} && make integrationtest K8SAPISERVERHOST=${openshift_api} DSPANAMESPACE=${DSPANAMESPACE} KUBECONFIGPATH=${KUBECONFIGPATH}
+    ${return_code}    ${output}    Run And Return Rc And Output    cd ${DSPO_SDK_DIR} && GIT_WORKSPACE=${DSPO_SDK_DIR} sh .github/scripts/tests/tests.sh --rhoai --k8s-api-server-host ${openshift_api} --kube-config ${KUBECONFIGPATH} --dspa-path ${DSPO_SDK_DIR}/tests/resources/dspa.yaml --external-dspa-path ${DSPO_SDK_DIR}/tests/resources/dspa-external.yaml --clean-infra --endpoint-type route
```

Comment:
The tests take a while to run. Could you try to use the `Run And Watch Command` keyword  in Common.robot so we'll be able to see the results in the Console in realtime? (in this or a follow-up PR)